libs/pilight/protocols/433.92/xiron.c

```diff
@@ -0,0 +1,221 @@
+/*
+	Copyright (C) 2014 CurlyMo
```

Comment:
Can you add yourself here?
Also change the license to MPLv2.

Reply:
Fixed it